website/content/en/docs/getting-started/_index.md

```diff
@@ -0,0 +1,236 @@
+commonly used values. 
+
+```bash
+export CLUSTER_NAME=$USER-karpenter-demo
+export AWS_DEFAULT_REGION=us-west-2
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+KARPENTER_VERSION=$(curl -fsSL \
+  https://api.github.com/repos/awslabs/karpenter/releases/latest \
+  | jq -r '.tag_name')
+```
+
+### Create a Cluster
+
+Create a cluster with `eksctl`. The [example configuration](eks-config.yaml) file specifies a basic cluster (name, region), and an IAM role for Karpenter to use. 
+
+```bash
+## This command won't work until a release after v0.2.8 is released
+curl -fsSL  https://raw.githubusercontent.com/awslabs/karpenter/"${KARPENTER_VERSION}"/pkg/cloudprovider/aws/docs/eks-config.yaml \
+  | envsubst \
+  | eksctl create cluster -f -
+```
+
+This guide uses a regular (un-managed) node group to host Karpenter.
+
+Karpenter itself can run anywhere, including on self-managed node groups, [managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html), or [AWS Fargate](https://aws.amazon.com/fargate/).
+
+Karpenter will provision new traditional instances on EC2. 
+
+Additionally, the configuration file sets up an [OIDC
+provider](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens),
+necessary for IRSA (see below). Kubernetes supports OIDC as a standardized way
+of communicating with identity providers. 
+
+### Setup Authentication from Kubernetes to AWS (IRSA)
+
+IAM Roles for Service Accounts (IRSA) maps Kubernetes resources to roles
+(permission sets) on AWS. 
+
+First, define a role using the template below. It provides full access to EC2,
+and limited access to other services such as EKS and Elastic Container Registry
+(ECR).
+
+```bash
+# Creates IAM resources used by Karpenter
+TEMPOUT=$(mktemp)
+curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/"${KARPENTER_VERSION}"/docs/aws/karpenter.cloudformation.yaml > $TEMPOUT \
+&& aws cloudformation deploy \
+  --stack-name Karpenter-${CLUSTER_NAME} \
+  --template-file ${TEMPOUT} \
+  --capabilities CAPABILITY_NAMED_IAM \
+  --parameter-overrides ClusterName=${CLUSTER_NAME}
+```
+
+Second, create the mapping between Kubernetes resources and the new IAM role. 
+
+```bash
+# Add the Karpenter node role to your aws-auth configmap, allowing nodes with this role to connect to the cluster.
+eksctl create iamidentitymapping \
+  --username system:node:{{EC2PrivateDNSName}} \
+  --cluster  ${CLUSTER_NAME} \
+  --arn arn:aws:iam::${AWS_ACCOUNT_ID}:role/KarpenterNodeRole-${CLUSTER_NAME} \
+  --group system:bootstrappers \
+  --group system:nodes
+```
+
+Now, Karpenter can send requests for new EC2 instances to AWS. 
+
+### Install Karpenter Helm Chart
+
+Use helm to deploy Karpenter to the cluster. 
+
+We created a Kubernetes service account when we created the cluster using
+eksctl. Thus, we don't need the helm chart to do that.
```

Comment:
Great. WDYT about having a command here (or a link to a command) to patch the serviceAccount for BYOC experiences or debugging purposes?

Reply:
i agree but i think that's out of scope for the PR right now.